src/graph/utils/sycl_check.hpp

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
```

Comment:
Is it intended to add this file?

Reply:
No, it was not. Thank you for pointing this out, I have removed the change. (I believe this might have been a earlier rebase artefact)